demo/vl-modal-auto-open.html

```diff
@@ -4,8 +4,10 @@
         <title>vl-modal voorbeeld</title>
         <script type="module" src="/src/vl-modal.src.js"></script>
         <script type="module" src="demo.js"></script>
-    
-         <script type="text/javascript">
+
+        <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-button/style.css"/>
```

Comment:
Het is niet de button stijl die ontbreekt, maar wel die van de vl-ui-core die sinds de major update naar 4.0.0 niet meer automatisch toegevoegd wordt aan de head tag. Dit wordt later vervangen door een vl-ui-body implementatie.